.github/swagger.yml

```
@@ -0,0 +1,41 @@
+name: Publish Swagger Specs
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up Cache
+        uses: actions/cache@v1
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
+          restore-keys: ${{ runner.os }}-gradle-
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Run Swagger Publisher
+        run: ./gradlew integration --tests uk.gov.hmcts.reform.waworkflowapi.config.SwaggerPublisherTest
+      - name: Commit to repository
+        run: |
+          mkdir swagger-staging
+            cd swagger-staging
+          git init
+          git config user.email "jenkins-reform-iac@hmcts.net"
+          git config user.name "WA Workflow API"
+          git remote add upstream https://${{ secrets.GH_TOKEN }}@github.com/hmcts/reform-api-docs.git
```

All repos have SWAGGER_PUBLISHER_API_TOKEN available to them

```suggestion
          git remote add upstream https://${{ secrets.SWAGGER_PUBLISHER_API_TOKEN }}@github.com/hmcts/reform-api-docs.git
```